docs/source/api/v2/deliveryservices_xmlid_xmlid_urlkeys.rst

```diff
@@ -21,12 +21,14 @@
 
 ``GET``
 =======
+.. seealso:: :ref:`to-api-deliveryservices-id-urlkeys`
+
 Retrieves URL signing keys for a :term:`Delivery Service`.
 
 .. caution:: This method will return the :term:`Delivery Service`'s **PRIVATE** URL signing keys! Be wary of using this endpoint and **NEVER** share the output with anyone who would be unable to see it on their own.
 
 :Auth. Required: Yes
-:Roles Required: None
+:Roles Required: "read-only"
```

Comment:
using "None" instead of "read-only" is pretty standard. Changing it would be an improvement, but I think it'd be better to keep consistency for now and change it everywhere in a different PR.

Reply:
Changed to None in 98a96c343c.